### PR TITLE
Bug #7587 - Adds javascript validation and strength feedback of password on ...

### PR DESCRIPTION
**Pull request by dragonfly9517**
...New User and Edit User screens

### CODE REVIEW

**Comment by dLobatog** [comment]
[test]

**Comment by dLobatog** [comment]
This is quite fancy actually :smile:  thanks!

It looks like this if anyone is curious: 

There's one problem that blocks this from merging in my opinion. When editing users, I always get the "User" label red by default because the password field is empty when you edit it. I suggest only loading the javascript when the user is actually writing to avoid this.

File: app/assets/javascripts/password_strength.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+//= require jquery_pwstrength_bootstrap
+function password_strength_feedback(password){
+  var options = {
+      common : {
+        onLoad : function(){
```
Comment (dLobatog) [inline app/assets/javascripts/password_strength.js]: Just change this `onLoad` to `onKeyUp` and this will be ready :smile:

**Comment by dLobatog** [comment]
@dragonfly9517  Oh sorry, one last thing. All your work has to be in one commit :white_check_mark: , and it has to start the title with "Fixes #7587". This is just so our CI system picks it up.

**Comment by dragonfly9517** [comment]
@eLobato  : Fixed the issue with red user labels on edit screen, and also merged everything into one commit. Thanks for your inputs.

**Comment by dLobatog** [comment]
[test]

**Comment by dLobatog** [comment]
Merged as https://github.com/theforeman/foreman/commit/9f2feee3f395d044f0daffad5950281cce4b74aa , thanks @dragonfly9517 !

**Pull request closed without merging**